tests/testnameresolve.nim

```diff
@@ -90,18 +90,6 @@ suite "Name resolving":
 
       check testOne("/ip6/::1/tcp/0", "/ip6/::1/tcp/0")
 
-    asyncTest "test multiple resolve":
```

Comment:
why is this removed?

Reply:
oh I see, you're removing `resolveMAddresses`... hmm, ok I guess it is a better interface overall since a name can resolve to multiple addrs.

Reply:
Yeah, we could wrap `resolveMAddress` in a `resolveMAddresses`, but we wouldn't use it in libp2p since we need to keep the original hostname for each original maddress